deploy/olm-catalog/knative-eventing-operator/0.6.0/eventing_v1alpha1_install_crd.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: installs.eventing.knative.dev
+spec:
+  group: eventing.knative.dev
+  names:
+    kind: Install
+    listKind: InstallList
+    plural: installs
+    singular: install
+    shortNames:
+    - kei
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            namespace:
```

Comment:
FYI, I'd like to remove `spec.namespace` and `status.resources`, but I'd like to merge this first. No sense in compounding changes in a single WIP. Unlike you, I do NOT prefer to drink from the firehose. ;)